lib/Sema/TypeCheckType.cpp

```diff
@@ -4128,6 +4126,18 @@ TypeResolver::resolveDeclRefTypeRepr(DeclRefTypeRepr *repr,
     return ErrorType::get(result->getASTContext());
   }
 
+  // Raise an error if anyone explicitly refers to the Copyable type, anywhere.
+  // We're not ready yet for anyone to express these kinds of constraints.
```

Comment:
Maybe just put an UnavailableAttr on it when you synthesize it?

Reply:
good idea. thanks! this works well.